themes/src/main/resources/theme/keycloak/welcome/theme.properties

```diff
@@ -1,7 +1,7 @@
-styles=css/welcome.css
 import=common/keycloak
 
-stylesCommon=node_modules/patternfly/dist/css/patternfly.css node_modules/patternfly/dist/css/patternfly-additions.css
+stylesCommon=node_modules/@patternfly-v5/patternfly/patternfly.min.css node_modules/@patternfly-v5/patternfly/patternfly-addons.css
```

Comment:
The fact that we're not loading PF5 styles could be a breaking change for users extending the `keycloak` theme.

Reply:
I think that's OK.